oars_gb_pkg/nodes/sensing/wind_filter.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+import rospy
+import numpy as np
+from collections import deque
+from std_msgs.msg import UInt8
+from geometry_msgs.msg import Pose2D
+
+'''
+This code subscribes to /sensors/airmar/wind/rel and /sensors/airmar/wind/true
+and stores on a rolling basis the most recent 10 values into a deque (or a rolling
+que). The code converts that deque into a numpy array and performs an
+average function on the array. The resulting values are a rolling average, and
+they are published to /weather/wind/rel and /weather/wind/true
+'''
+
+# Create deques to store last 10 values
+wind_rel_deque = {'x': deque(maxlen=10), 'theta': deque(maxlen=10)}
+wind_true_deque = {'x': deque(maxlen=10), 'theta': deque(maxlen=10)}
+
+# Create publishers for relative and true wind speed topics
+pub_rel = rospy.Publisher('/weather/wind/rel', Pose2D, queue_size=10)
+pub_true = rospy.Publisher('/weather/wind/true', Pose2D, queue_size=10)
+
+def airmar_subscriber():
+    rospy.init_node("airmar_wind_filter", anonymous=True) # Initialize nodes
```

Comment:
Can you please create a class called `WindFilter` that initializes the node in its `__init__` method?